blamer.el

```diff
@@ -992,6 +992,12 @@ LOCAL-TYPE is force replacement of current `blamer-type' for handle rendering."
   (remove-hook 'post-command-hook #'blamer--try-render t)
   (remove-hook 'window-state-change-hook #'blamer--try-render t))
 
+(defun blamer--get-user-name ()
+  "Query git config for user name"
```

Comment:
```suggestion
  "Query git config for user name."
```
It raised linter error `First sentence should end with punctuation (emacs-lisp-checkdoc)`